### ISSUE TITLE
Using ONNX model in c# Windows Form Application

### ISSUE THREAD

**Issue by amuthalingeswaranbose** [migueldeicaza/TensorFlowSharp#455]
**Is your feature request related to a problem? Please describe.**
I have trained a deep learning model of Mask RCNN in Keras and have derived a OnnX model (weight matrix) which is able to run and test images in Python successfully. I need to use the same ONNX model in Windows form Application, C# through visual studio.

**System information**
Windows 10 Enterprise 2016 LTSB
Visual Studio 2019 
**ONNX Runtime version (you are using):**
Mask RCNN, in ONNX Runtime v0.5.0

**Describe the solution you'd like**
I should be use the model and test images in Windows form application, C#.

**Comment by gmlwns2000** [comment]
Hi,

This is not about TFSharp, but if you run your ONNX model in latest Windows, I recommend to use Windows ML.

Here is [MSDN page](https://docs.microsoft.com/ko-kr/windows/ai/windows-ml/integrate-model) about to run ONNX. 

I tried it few days ago, it was really easy to work, except GPU runner. If you need GPU you have to write GPU tensorization, but it seems to be easy too.

Here is [my implements](https://github.com/NeuralAction/Vision/blob/9b50a72e4c8c560702849e2b0135acbe6e140c16/Vision/Detection/EyeGazeDetector.cs#L57), and I will work on GPU in soon too.
